src/tagmanager.cpp

```diff
@@ -39,13 +39,26 @@ HSTag* TagManager::find(const string& name) {
     return {};
 }
 
+//! if the name is a valid tag name, return "", otherwise return an error message
+string TagManager::isValidTagName(string name) {
+    if (name.empty()) {
+        return "An empty tag name is not permitted";
+    }
+    if (find(name)) {
+        return "A tag with the name \"" + name + "\" already exists";
+    }
+    if (find_monitor_by_name(name.c_str())) {
```

Comment:
Just one more thing... what's the purpose of this? Am I missing something or is this just not doing anything?

Reply:
Looks like a bad copy&paste mistake from monitormanager. This shouldn't be here, and I've removed this! Thanks!